README.md

```diff
@@ -36,10 +41,10 @@ We have released the GDK for Unreal this early in development because we want yo
 ## Where to get the GDK and starter projects
```

Comment:
starter projects .<- Didn't you want to rename this title to something which didn't say "starter project"?

Reply:
sample projects